xpublish-exploring/demo.py

```diff
@@ -26,9 +31,9 @@ def get_dataset(self, dataset_id: str):
 rest = Rest({})
 rest.register_plugin(TutorialDataset())
 
-# from mean import MeanPlugin
-# from lme import LmeSubsetPlugin
-# rest.register_plugin(MeanPlugin())
-# rest.register_plugin(LmeSubsetPlugin())
+from mean import MeanPlugin
+from lme import LmeSubsetPlugin
+rest.register_plugin(MeanPlugin())
+rest.register_plugin(LmeSubsetPlugin())
```

Comment:
I was thinking that we start with these commented out to keep things simpler with just a dataset provider plugin. Then uncomment and show `MeanPlugin` to demo a single new route. 

Finally uncomment and explore `LmeSubsetPlugin` to show how plugins can reuse and compose other plugins.

Reply:
I could go either way on this. It seemed like some people just wanted to make it work. Not that big a deal.